### PR TITLE
RDM-12714 Enable Global Search Index Creation/Recreation from AdminWeb

### PR DESCRIPTION
**Pull request by markdathornehmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/RDM-12714

### Change description ###
Provide an endpoint which will trigger creation/recreation of global search indices

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by markdathornehmcts** [comment]
Please review the changes so far but don't merge. Need pointers on unit tests & code coverage.

**Comment by markdathornehmcts** [comment]
Ready for review of the changes made

File: src/main/routes/routerGlobalSearch.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import * as express from "express";
```
Comment (danlysiak) [inline src/main/routes/routerGlobalSearch.ts:1]: Calling this file `indexGlobalSearch.ts` might be more consistent?

File: src/test/routes/indexGlobalSearch.spec.ts [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      const apiCall = mock("http://localhost:4451")
+        .post(GLOBAL_SEARCH_POST_ENDPOINT)
+        .reply(201, "Created");
+
+      return request(app)
+        .post(GLOBAL_SEARCH_POST_ENDPOINT)
+        .set("Cookie", "accessToken=ey123.ey456")
+        .then((res) => {
+          expect(apiCall.isDone()).to.be.false;
+          expect(res.status).to.equal(403);
```
Comment (danlysiak) [inline src/test/routes/indexGlobalSearch.spec.ts:117]: Can we add the verification checks on the error for this and the other 403 in the next test please, similar to the other test classes.

**Review by danlysiak — COMMENTED** [review]
Looks good to me, very minor comments

**Review by danlysiak — APPROVED** [review]

**Pull request merged**